### PR TITLE
W-11999384 ssl cert toggle change bsj

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 1]
```diff
@@ -47,14 +47,15 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
 == Create a Monitor
+
 To create a monitor:
 
 . Click *Create Monitor* in the *Functional Monitoring* section of Anypoint Monitoring.
-. Under *Step 1 - Setup the monitor*, specify a monitor name. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+. Under *Step 1 - Setup the monitor*, specify a monitor name. 
 . Click the *Step 2 - Select endpoints* tab.
-. Populate the fields with the information gathered in "Before You Begin."
+. Populate the fields for each endpoint with the information gathered in "Before You Begin." If the monitor needs to use self-signed certifications for an endpoint, toggle *Validate SSL Certificate*.
```
Comment (florvimberg) [inline modules/ROOT/pages/afm-create-monitor.adoc]: The toggle is turned on by default. By turning it off is that users allow unsafe ssl. I thought maybe we could change the text to something like:
_If the monitor needs to allow unsafe self-signed certifications for an endpoint, toggle *Validate SSL Certificate* off._

Also, we could write down that the default value is toggle ON which means unsafe validations OFF.

File: modules/ROOT/pages/afm-modify-monitor.adoc [hunk 2]
```diff
@@ -33,13 +34,14 @@ After following either method, the following options appear:
 
 * Tab one (*Step 1 - Select endpoints*) enables you to modify the endpoints.
 + 
-You can edit endpoint methods, URLs, optional headers, and assertions. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+You can edit endpoint methods, URLs, optional headers, and assertions. If the monitor needs to use self-signed certifications for an endpoint, toggle *Validate SSL Certificate*.
```
Comment (florvimberg) [inline modules/ROOT/pages/afm-modify-monitor.adoc]: Same here

**Review by florvimberg — DISMISSED** [review]
👍

File: modules/ROOT/pages/afm-create-monitor.adoc [hunk 3]
```diff
@@ -47,14 +48,15 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
...
 == Create a Monitor
+
 To create a monitor:
 
 . Click *Create Monitor* in the *Functional Monitoring* section of Anypoint Monitoring.
-. Under *Step 1 - Setup the monitor*, specify a monitor name. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+. Under *Step 1 - Setup the monitor*, specify a monitor name. 
 . Click the *Step 2 - Select endpoints* tab.
-. Populate the fields with the information gathered in "Before You Begin."
+. Populate the fields for each endpoint with the information gathered in <<before-you-begin>>. By default the monitor validates self-signed certifications for each endpoint, so *Validate SSL Certificate* is toggled on. If you want to allow unsafe self-signed certifications for an endpoint, toggle *Validate SSL Certificate* off.
```
Comment (dhirsch1) [inline modules/ROOT/pages/afm-create-monitor.adoc]:
```suggestion
. Populate the fields for each endpoint with the information gathered in <<before-you-begin>>. By default, the monitor validates self-signed certifications for each endpoint, so *Validate SSL Certificate* is toggled on. To allow unsafe self-signed certifications for an endpoint, toggle *Validate SSL Certificate* off.
```

Comment (dhirsch1) [reply modules/ROOT/pages/afm-create-monitor.adoc]: I think "toggled on" and "toggled off" are OK. You could use the terms "enabled" and "disabled" instead if you want to.

File: modules/ROOT/pages/afm-modify-monitor.adoc [hunk 4]
```diff
@@ -33,13 +34,14 @@ After following either method, the following options appear:
 
 * Tab one (*Step 1 - Select endpoints*) enables you to modify the endpoints.
 + 
-You can edit endpoint methods, URLs, optional headers, and assertions. Additionally, toggle *Validate SSL Certificate* to specify if the monitor needs to use self-signed certifications.
+You can edit endpoint methods, URLs, optional headers, and assertions. By default the monitor validates the self-signed certifications for each endpoint, so *Validate SSL Certificate* is toggled on. If you want to allow unsafe self-signed certifications for an endpoint, toggle *Validate SSL Certificate* off.
```
Comment (dhirsch1) [inline modules/ROOT/pages/afm-modify-monitor.adoc]:
```suggestion
You can edit endpoint methods, URLs, optional headers, and assertions. By default, the monitor validates the self-signed certifications for each endpoint, so *Validate SSL Certificate* is toggled on. To allow unsafe self-signed certifications for an endpoint, toggle *Validate SSL Certificate* off.
```

**Review by dhirsch1 — DISMISSED** [review]

**Review by dhirsch1 — DISMISSED** [review]

**Review by glenn-rodgers-sf — APPROVED** [review]

**Pull request merged**